src/common/icons/index.js

```diff
@@ -38,6 +38,21 @@ import gridIcon from "./grid"
 import boxIcon from "./box"
 import databaseIcon from "./database"
 import phoneIcon from "./phone"
+import doctorDashboardIllustrator from "./doctorDashboardIllustrator"
+// import microscopeIcon from "./microscope"
+// import weightLifterIcon from "./weightLifter"
```

Comment:
If you don't need this, don't comment it, delete it.

Reply:
i have deleted this comments @agustinustheo